app.js

```diff
@@ -18,9 +23,26 @@ database.on('error', (error) => {
 database.once('connected', () => {
   console.log('Database Connected')
 })
+initializePassport(passport)
+const generateSecretKey = () => {
+  return crypto.randomBytes(32).toString('hex')
+}
+
+const secretKey = generateSecretKey()
```

Comment:
SecretKey should be consistent between different executions of app. Dont generate use fixed.